pkg/engine/templates_generated.go

```diff
@@ -47824,13 +47824,13 @@ var _bintree = &bintree{nil, map[string]*bintree{
 			"1.9": {nil, map[string]*bintree{
 				"kubernetesmasteraddons-metrics-server-deployment.yaml": {k8sAddons19KubernetesmasteraddonsMetricsServerDeploymentYaml, map[string]*bintree{}},
 			}},
-			"antrea.yaml":                        {k8sAddonsAntreaYaml, map[string]*bintree{}},
-			"azure-cni-networkmonitor.yaml":      {k8sAddonsAzureCniNetworkmonitorYaml, map[string]*bintree{}},
-			"azure-policy-deployment.yaml":       {k8sAddonsAzurePolicyDeploymentYaml, map[string]*bintree{}},
-			"coredns.yaml":                       {k8sAddonsCorednsYaml, map[string]*bintree{}},
-			"dns-autoscaler.yaml":                {k8sAddonsDnsAutoscalerYaml, map[string]*bintree{}},
-			"ip-masq-agent.yaml":                 {k8sAddonsIpMasqAgentYaml, map[string]*bintree{}},
-			"kubernetesmaster-audit-policy.yaml": {k8sAddonsKubernetesmasterAuditPolicyYaml, map[string]*bintree{}},
+			"antrea.yaml":                                                  {k8sAddonsAntreaYaml, map[string]*bintree{}},
+			"azure-cni-networkmonitor.yaml":                                {k8sAddonsAzureCniNetworkmonitorYaml, map[string]*bintree{}},
+			"azure-policy-deployment.yaml":                                 {k8sAddonsAzurePolicyDeploymentYaml, map[string]*bintree{}},
+			"coredns.yaml":                                                 {k8sAddonsCorednsYaml, map[string]*bintree{}},
+			"dns-autoscaler.yaml":                                          {k8sAddonsDnsAutoscalerYaml, map[string]*bintree{}},
+			"ip-masq-agent.yaml":                                           {k8sAddonsIpMasqAgentYaml, map[string]*bintree{}},
+			"kubernetesmaster-audit-policy.yaml":                           {k8sAddonsKubernetesmasterAuditPolicyYaml, map[string]*bintree{}},
 			"kubernetesmasteraddons-aad-default-admin-group-rbac.yaml":     {k8sAddonsKubernetesmasteraddonsAadDefaultAdminGroupRbacYaml, map[string]*bintree{}},
```

Comment:
Just a kind remind,  it seems that the code generated in your env is not go-formated @AbelHu

Reply:
interesting. I also run `make generate`